nrf_security/doc/glue_layer.rst

```diff
@@ -30,8 +30,8 @@ This function is called from the API which provides configuration changes that d
     }
 
 In this example, the AES CCM support in the backend will report priority level 3 if the key size is 128, or 0 if the key size is different.
-The :ref:`nrf_security_backends_cc3xx` does not support a larger key size.
-If the key size is larger than 128 bits, then another enabled backend is used.
+Level 3 has higher priority than 0.
```

Comment:
This text wasn't that well written in the first place. 0 implies not supported (i.e. try the next backend)